HW_1/Task2.py

```diff
@@ -0,0 +1,19 @@
+'''Напишите код, который запрашивает число и сообщает 
+является ли оно простым или составным. Используйте 
+правило для проверки: “Число является простым, если делится 
+нацело только на единицу и на себя”. Сделайте ограничение на ввод 
+отрицательных чисел и чисел больше 100 тысяч.'''
+
+user_num = int(input('Введите положительное число не больше 100К: '))
+count = 0
+
+if user_num < 0 or user_num > 100000:
+    print('Ошибка, введите другое число.')
+else:
+    for i in range(1, user_num + 1):
+        if user_num % i == 0:
+            count += 1
+    if count == 2:
+        print('Число является простым.')
+    else:
+        print('Число является составным.')
```

Comment:
Зачем нужна переменная `count`, если при существовании хотя бы одного делителя, не являющимся этим числом единицей, то число уже будет составным? Да, и первое условие, на мой взгляд можно переделать.
```suggestion
if 0 <= user_num <= 100000:
    for i in range(1, user_num + 1):
        if user_num % i == 0:
            print('Число является составным.')
            break
    
    else:
        print('Число является простым.')

else:
     print('Ошибка, введите другое число.')
```